Project/register.js

```diff
@@ -0,0 +1,177 @@
+/*
+register.js
+J O'Regan
+15/11/2019
+*/
+
+var surveyArr = [];
```

Comment:
Example comment: maybe add a comment explaining what the surveyArr array does

Reply:
maybe add a comment explaining what the surveyArr array does?